config/initializers/wfs_rails.rb

```diff
@@ -1,6 +1,6 @@
-if Settings.WFS_RAILS.ENABLE
-  booted_url = Capybara::Discoball::Runner.new(OpenStruct.new(new: WfsRails::Engine.app)).boot
-  # Reconfigure Dor::Config.workflow.client with booted_url of WfsRails
-  Dor::Config.workflow.url = booted_url
-  Settings.WORKFLOW_URL = booted_url
-end
+# if Settings.WFS_RAILS.ENABLE
```

Comment:
Is there a reason to keep this file if it's just a big comment?

Reply:
Naw, I just wanted to see if this was going to work before cleaning it up a bit.

Reply:
WFM, Just wanted to check it out. :+1: from me here.